_config.yml

```diff
@@ -4,9 +4,13 @@ version: 0.14.0
 permalink: pretty
 google_analytics_ua: UA-48605964-43
 
+encoding: utf-8
```

Comment:
@shawnbot this is what I was referring to in Slack. An error occurs without this explicitly set. My local Jekyll version is set to `jekyll 3.2.1`.